apps/netsuite/connector.py

```
@@ -1692,6 +1693,9 @@ def post_expense_report(
             created_expense_report = self.connection.expense_reports.post(expense_report_payload)
             return created_expense_report
 
+        except zeep_exceptions.Fault as exception:
```

need not handle here, handle it in netsuite/exceptions, this will silence the exception and it might end up in IN_PROGRESS